packages/styles/src/components/slider/variants.tsx

```diff
@@ -0,0 +1,29 @@
+// SPDX-FileCopyrightText: 2024 Deutsche Telekom AG
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { tv } from "tailwind-variants"
+
+export const sliderVariants = tv({
+	slots: {
+		sliderRoot: "flex flex-col w-full",
+		sliderHeader: "flex justify-between",
+		sliderTrack:
+			"w-full h-8 before:absolute before:inset-x-0 before:top-1/2 before:-translate-y-1/2 before:h-1 before:bg-border before:rounded-full",
+		sliderThumb: "h-6 w-6 bg-primary rounded-full top-1/2",
```

Comment:
Disabled state seems to be missing, Slider should have styling specific to when he is disabled